### PR TITLE
Add Dockerfile to project

### PR DESCRIPTION
**Pull request by eze8789**
A simple Dockerfile exposing both TCP and HTTP ports to use as needed, since configuration is managed with a JSON file any change to it will need a rebuild of the image.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM golang:1.14-alpine as builder
```
Comment (tomiok) [inline Dockerfile]: 1.14 is the latest for alpine?

**Comment by tomiok** [comment]
@eze8789 nice work. Could you add some examples of how to run it as a docker service?

**Review by tomiok — CHANGES_REQUESTED** [review]
Add a reference for how to run the docker image (and build) in the README.md

File: Dockerfile (hunk 1 above)
Comment (eze8789) [reply Dockerfile]: Bumping to latest version for the builder

**Comment by eze8789** [comment]
> Add a reference for how to run the docker image (and build) in the README.md

Added the details to the README

**Review by tomiok — APPROVED** [review]

**Pull request merged**